source/index.html.md

```diff
@@ -136,54 +144,549 @@ https://oauth.molofinance.com/token/
+        "updated_at": "2019-10-18T14:20:41.747390Z"
+    },
+    {
+        "id": 226,
+        "status": "valid",
+        "mortgage_type": "new_purchase",
+        "borrower_type": "individual_borrower",
+        "created_by": 292,
+        "created_at": "2018-05-17T16:56:06.290373Z",
+        "updated_at": "2019-10-18T14:20:19.535138Z"
+    },
+    ...
+]
+```
 
+**Send a `GET` request to our applications endpoint to retrieve a list of past applications.**
 
+[https://partner.molofinance.com/api/v1/applications/](https://partner.molofinance.com/api/v1/applications/)
 
 
-# API
-The Molo partner API can be used by any partner who has obtained an access token.
-You can access these endpoints by using the `bearer token` authentication header and entering this token as the value.
+## Retrieve details of a specific application
 
-## Mortgage applications
 ```shell
 curl -X GET \
-  https://partner.molofinance.com/v1/applications/ \
-  -H 'Authorization: Bearer <ACCESS TOKEN>' \
-  -H 'cache-control: no-cache'
+    https://partner.molofinance.com/api/v1/applications/ \
+    -H 'Accept: */*' \
+    -H 'Authorization: Bearer <TOKEN>' \
+    -H 'Content-Type: application/json' \
+    -H 'Host: partner.molofinance.com' \
+```
 
+> **Expected response: Status 200**
 
+```json
+{
+    "id": 1000002171,
+    "status": "pending_product_selection",
+    "mortgage_type": "new_purchase",
+    "borrower_type": "individual_borrower",
+    "created_by": 1387,
+    "created_at": "2019-05-15T13:05:41.479039Z",
+    "updated_at": "2019-10-18T14:20:41.747390Z",
+    "desired_loan_amount": "66000.00",
+    "property_value": 110000.0,
+    "monthly_rent": 1100.0,
+    "loan_term": 5,
+    "applicants": [
+        3548,
+        3542
+    ]
+}
 ```
+
+**Send a `GET` request to our applications endpoint to retrieve details of a specific application.**
+
+[https://partner.molofinance.com/api/v1/applications/{pk}/](https://partner.molofinance.com/api/v1/applications/{pk}/)
+
+
+## Get a decision in principal
+
+```shell
+curl -X POST \
+    http://partner.molofinance.com/api/v1/applications/<APPLICATION_ID>/get-dip/ \
+    -H 'Accept: */*' \
+    -H 'Authorization: Bearer <TOKEN>' \
+    -H 'Content-Type: application/json' \
+    -H 'Host: partner.molofinance.com' \
+    -d '{' \
+            '"dip_consent": "true"', \
+       '}'
+```
+
+> **Expected response: Status 200**
+
+```json
+{
+    "decision": "valid"
+}
+```
+
+**Send a `POST` request to our applications dip endpoint to get a decision in principal.**
+
+[https://partner.molofinance.com/api/v1/applications/{pk}/get-dip/](https://partner.molofinance.com/api/v1/applications/{pk}/get-dip/)
+
+You must send a the parameter:
+
+- `dip_consent` A boolean, you must send "true" to get a dip
+
+The response will be in the form:
+
+- `dip_agreed` Whether Molo can give you a decision in principal. A boolean.
+- `link_to_dip` A link to the DIP itself so that it can be downloaded.
```

Comment:
I think it's outdated

Reply:
we'll shore up the details later, I _think_ this is going to be a base64 field, but I'll confirm with Sergey tomorrow. Mojo need this on Dev today though

Reply:
`get-dip` API endpoint returns currently just a `decision` field

Reply:
yes, but that'll change.
Mojo aren't going to be using this yet, they're just going to use it to estimate their workload